.github/actions/setup-install/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Setup and Install
+description: Setup nodejs and install npm packages
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup Node.js and Cache
+      uses: actions/setup-node@v4
+      with:
+        node-version: 18
```

Comment:
Could we bump it to version 20?

Reply:
Done. Let's see if it is stable enough,  since it's too new.